package.json

```
@@ -43,7 +43,7 @@
     "percy": "percy exec -- node snapshots.js",
     "icon-svgs-to-mixins": "node scripts/convert-svgs-to-icon-mixins.js icons"
   },
-  "version": "2.35.0",
+  "version": "2.37.0",
```

Why it became 37 out of the sudden? We just released 2.35, so this will be 2.36

```suggestion
  "version": "2.36.0",
```

---

Sorry, mistake when rebasing from your release. Fixed.